internal/providers/github_provider.go

```diff
@@ -35,16 +35,18 @@ type GitHubEnv struct {
 	Repository string `env:"GITHUB_REPOSITORY"`
 }
 
+type EventPayloadData struct {
```

Comment:
😻

Reply:
💭 I might name `GitHubEventPayloadData` because it's global in the providers package

Reply:
Good call -- I'll make that change.